test/support/utils.h

```diff
@@ -42,6 +42,12 @@
 #    include "utils_sycl.h"
 #endif
 
+#if __has_include(<sycl/sycl.hpp>)
+#    define THE_SYCL_HPP <sycl/sycl.hpp>
+#else
+#    define THE_SYCL_HPP <CL/sycl.hpp>
+#endif
```

Comment:
Let's include `sycl` header inside `utils_sycl.h`. If we do so, then this header will be automatically included by overwhelming majority of the tests (without adding `#include THE_SYCL_HPP` lines)

Reply:
The macros THE_SYCL_HPP was already deleted in last commits because it's not required anymore.